### DISCUSSION TITLE
What does `DEFAULT_LOCALE` of umami 2.4.0 do?

### DISCUSSION THREAD

**Discussion by aaro-n** [umami-software/umami#2241] [Q&A]
Does `DEFAULT_LOCALE` set the default display language of umami? I did not find the description of this variable in the documentation. If it is to set the default display language of umami, how does umami set it to display Chinese? `DEFAULT_LOCALE = "zh-CN"`This setting is invalid, umami still displays English by default.

**Comment by liiijz** [comment]
same here
